installer-howto/installer-howto.rst

```diff
@@ -0,0 +1,246 @@
+.. _installer_howto:
+
+Rockstor’s “Built on openSUSE” installer - Beta release
+=======================================================
+
+.. _installer_thanks:
+
+Thanks
+------
+
+Rockstor’s developers would like to thank the entire OSS community for making any of this possible.
+Specifically with our fledgling move / rebase on openSUSE Leap15.2 our thanks go to the openSUSE/SuSE organisations
+and the larger community for the extreme generosity we all benefit from in a myriad of seen and unseen ways.
+
+**Please note: Rockstor is in no way affiliated with openSUSE / SuSE and does not wish, under any circumstances, to misrepresent these organisations.**
+*Any errors you encounter are most likely our fault and we try, on a current volunteer basis, to support our users via our* `friendly forum: <https://forum.rockstor.com/.>`_.
+
+It is hoped that Rockstor’s ongoing small contribution back to the larger community is of value,
+and if this is so for you please consider either subscribing to our Stable Channel updates or taking part in our developer orientated Testing Channel.
+Details of these Rockstor package update services are available in our `Official Docs <http://rockstor.com/docs/>`_ subsection `Update Channels <http://rockstor.com/docs/update-channels/update_channels.html>`_.
+Rockstor’s ongoing development is wholly dependent on community support via either of these channels and by positively participating in our forum, linked above.
+It is hoped that in time we can contribute back more significantly, and possibly financially, as our own community and Stable Channel subscriptions grow.
+Also note that irrespective of Rockstor update channel selection or otherwise, upstream updates are available by normal command line means and via our Web-UI.
+See :ref:`dash_and_updates` later in this guide for the details.
+
+.. _our_kiwi_ng_installer:
+
+Our "Built on openSUSE" Wiki-ng created installer
```

Comment:
```suggestion
Our "Built on openSUSE" Kiwi-ng created installer
```

Reply:
@FroggyFlox Thanks, you wouldn't believe the number of times I keep doing this.